.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build degitx-simulator
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  diktat_check:
```

Comment:
@IlyaMoskva I just missed this name: let's fix this job name, now Github shows check named `diktat_check` in CI list

Reply:
Haha )) Thanks, @g4s8 , fixed as well.